packages/devtools/lib/src/debugger/debugger_state.dart

```diff
@@ -292,3 +290,18 @@ class DebuggerState {
     _breakpoints.add(isolate.breakpoints);
   }
 }
+
+class _Property<T> {
+  _Property([T initialValue]) : _value = initialValue;
+
+  T _value;
+  T get value => _value;
+
+  final _controller = StreamController<T>.broadcast();
+  Stream<T> get stream => _controller.stream;
```

Comment:
I think the code using `BehaviorSubject` is depending on the behavior that when you first add yourself as a listener, you get the current value as an event. That's how we sync the UI up to the initial values. We should likely preserve that behavior here.

Reply:
I'm handling this manually in the calling code where I duplicate the behavior of the on change and perform it synchronously. I suspect some of this isn't necessary, and the tests don't fail without it, but I'm not completely sure. I wasn't able to find behavior regressions without it either, but I don't know the app that well.